tests/helpers/test_network.py

```diff
@@ -273,15 +256,15 @@ async def test_get_cloud_url(hass: HomeAssistant):
         "async_remote_ui_url",
         return_value="https://example.nabu.casa",
     ):
-        assert _async_get_cloud_url(hass) == "https://example.nabu.casa"
+        assert _get_cloud_url(hass) == "https://example.nabu.casa"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)